docs/resources/client/textures/atlases.md

```diff
@@ -0,0 +1,98 @@
+# Texture Atlases
+
+Many textures in Minecraft are compiled within Atlases (also known as sprite sheets) for optimisation purposes. Most of them are automaticaly generated by Minecraft when registered throught the use of datapacks or [Registries][reg] (such as block textures).
+
+## Texture Atlas Sources
+
+Texture atlas sources are json files located inside the `atlases` asset folder.
+A texture atlas source describes the name of the targeted atlas and its contents. Additions to the atlases need to be created under the namespace of a mod or datapack.
```

Comment:
Additions to what atlas? Are you talking about creating your own atlas here, or adding to an existing atlas?